### DISCUSSION TITLE
What video cards do you have asking developers of 86box

### DISCUSSION THREAD

**Discussion by WhiteMagicRaven** [86Box/86Box#1933] [General]
I thinking of send some video cards to 86box developers, but first of all i want to know what video cards you already have.
I interested in:
Matrox Impression
Matrox Millenium
Cirrus Logic gd5462 and gd5464
Permedia NT (with IBM DAC)
Creative 3D Blaster VLB
Creative 3D Blaster PCI
Paradise Tasmania 3D
Nvidia NV1 (card series)
ATI 3D Rage
Rendition Verite
PowerVR PCX2 or other
S3 Savage3D or Savage 4

if there is possible , lets go into PM 
or by mail my is [mynicknamehere][at]gmail[dot]com

not a video card but also interested in 
Real Magic Mpeg decoder or Reel Magic

**Comment by ppgrainbow** [comment]
I was thinking about the MCA-based 8514/A, XGA and XGA-2 graphic cards that will work on emulated PS/2-based machines with a MCA bus and the ISA-based Tseng Labs ET3000.

**Comment by OBattler** [comment]
I currently have no old graphics card, nor do I have an old PC capable of using one, not that my main PC isn't itself old already (but not *that* old), being a Dell workstation from 2009. So far, any real hardware probing I done, was either on what scraps I could access on my PC, and mostly, relying on other people to probe stuff for me.

**Comment by WhiteMagicRaven** [comment]
ok in other way:
i have retro pc with win 98 , so how i can organize remote access ? , so you can figure out how one or another video card works . 
or if you need help on testing something i can do it.
currently i have : 
Matrox Impression
Matrox Millenium
Cirrus Logic gd5464
ATI 3D Rage
Rendition Verite
PowerVR PCX2
Savage 4

**Reply by OBattler** [reply]
That's a nice selection. I do have in my plans to eventually do the Cirrus Logic CL-GD 5464 (at least that's the easiest for me to of these, the rest will have to wait a bit more), so any help you can provide, is going to be useful.

You can also help with things other than graphics cards, for example, what keyboard controller does your PC have?

**Comment by WhiteMagicRaven** [comment]
ps/2 and usb, also have com mouse

**Reply by BurnedPinguin** [reply]
Hey there! At OBattler's initiative, do you mind running something on your Windows 98 machine? 

Attached is a ZIP file which you must extract in an empty folder on your machine. Then, you must boot into real mode DOS (F8 on startup -> Safe mode command prompt), and then run the batch file that you extracted. You will be presented with a list of options, start by pressing 1, and continuing when prompted to dump something else. You will be sent back to that same list, and then press number 2 for option 2, and so on for options 1, 2, 3, and 4.
[86BXDUMP.zip](https://github.com/86Box/86Box/files/7703924/86BXDUMP.zip)
?

**Comment by WhiteMagicRaven** [comment]
For start i dumped Matrox Impression
https://raven-05.narod.ru/dumps/impression/dumps.zip

**Reply by Torinde** [reply]
Matrox Impression registers [http://www.vgamuseum.info/index.php/component … 61f8101f4685b4b](http://www.vgamuseum.info/index.php/component/k2/item/download/291_6c45b4e53dacd49de61f8101f4685b4b)

**Comment by WhiteMagicRaven** [comment]
[dumps.zip](https://github.com/86Box/86Box/files/7706845/dumps.zip)

**Comment by WhiteMagicRaven** [comment]
[dumps1.zip](https://github.com/86Box/86Box/files/7737737/dumps1.zip)

Finally dumped every video card mentioned by me here

and photos for reference, will be available 7 days. from moment of posting, request if need reupload
https://fex.net/ru/s/yklnsr4

**Comment by WhiteMagicRaven** [comment]
What else i can do)?

i found that there a pcipasstrough 
https://www.vogons.org/viewtopic.php?p=1008026#p1008026

so its possible to run a Virtual Machine in newer OS'es and pass trough to real card (full access i guess, because indy car 2 is dos and it working) it means i can give you access to my PC by team viewer , anydesk or other apps like this, and you can do experiements inside of Virtual Machine to figure out how cards working. 

I also seen there is AGP to PCI converter , so AGP also possible =)

**Comment by WhiteMagicRaven** [comment]
Here you can find many video card defenitions
https://github.com/q3k/Cucumber/blob/master/Kernel/include/Alentours/PCIDB.h

Here you can see incomplete 3d rage emulation
https://github.com/dingusdev/dingusppc/blob/master/devices/video/atirage.cpp

In mame couple files about matrox millennium emulation attempt (search by "matrox", "mga2064")
i found files , SDL_DirectFB_modes.c , video.lua ,mess.lua , mga2064w.cpp  , mga2064w.h , pcipc.cpp
 
In mame couple files about PowerVR2 emulation, its equvalent of "Neon250	PowerVR 250PC	125 MHz	2D/3D PC Add-in Board" for PC, the PowerSGL2 is compatible 50% to PowerSGL (the PCX1 and PCX2 cards), also there is compatibility layer that allows to support more PowerSGL (PCX2) games on PowerSGL2 video card the Nuon250

**Reply by WhiteMagicRaven** [reply]
Neon 250 the PowerVR2 (PowerSGL2) also emulated in Dreamcast, asked others videocard collectors, maybe they also willing to help... 
...

**Reply by WhiteMagicRaven** [reply]
Also dreaming about KVM it gives remote acess to pc with full control including entering to bios. with that i can give you access to that specific hardware

**Comment by WhiteMagicRaven** [comment]
New dumps of: (by Ligeti Endre)
PowerVR Neon-250
Diamond Fire GL 1000
Diamond Edge 3D 3240
Diamond Edge 3D 2200
https://seldon.ipdns.hu/86BXDUMPS_1.zip
https://raven-05.narod.ru/86BXDUMPS_1.zip

**Comment by WhiteMagicRaven** [comment]
https://github.com/powervr-graphics/PowerVR-Series1

PowerVR video accelerators is released their source codes

**Comment by Torinde** [comment]
Related discussion: https://github.com/joncampbell123/dosbox-x/issues/291, https://github.com/86Box/86Box/discussions/4555

Rendition Verite: https://github.com/dosbox-staging/dosbox-staging/discussions/3156

**Reply by Torinde** [reply]
Rendition Verite support is progressing well:
- [Source for the DOSbox Staging part](https://sourceforge.net/projects/dosbox-staging-rendition/)
- [Source for RRedline1to2](https://sourceforge.net/projects/rready-v2k/)
- [Website](https://nirvtek.com/rready.php)
- [Discussion and Free binary?](https://www.vogons.org/viewtopic.php?t=95721)
- [Paid Microsoft Store app](https://apps.microsoft.com/detail/9nrcqj8vlgtk?hl=en-us&gl=US)
- planning for CGL (Creative Graphics Library) support

**Comment by WhiteMagicRaven** [comment]
Hello accidentally i found this web site 
https://bitsavers.trailing-edge.com/components/

this have 

(matrox folder) Matrox Impression Datasheet (registers here)
(ati folder) Ati 3d Rage documents , registers also.
(s3 folder) s3 savage4 documentation registers etc.
(3dfx folder) voodoo 4 and 5 some documentations .

**Comment by WhiteMagicRaven** [comment]
https://www.vgamuseum.info/index.php/cpu/item/374-s3-savage3d
https://www.vgamuseum.info/index.php/cpu/item/376-s3-savage4-lt
https://www.vgamuseum.info/index.php/cpu/item/375-s3-savage4-pro
https://videowww.vgamuseum.info/index.php/benchmarks/diag-text/item/578-diamond-stealth-iii-s540-s3-savage4-pro
https://www.vgamuseum.info/index.php/cpu/item/340-rendition-verite-v2100
https://vgamuseum.info/images/doc/videologic/powervr_technicalback.pdf
https://www.vgamuseum.info/index.php/cpu/item/950-ati-3d-rage
https://vgamuseum.info/index.php/home/item/97-ati-3d-rage-pro-pci
https://videowww.vgamuseum.info/index.php/technologies/item/542-diamond-edge-3d-2120xl-nvidia-nv1
https://www.vgamuseum.info/index.php/f-naq/item/1166-cirrus-logic-cl-gd5462
https://www.vgamuseum.info/index.php/f-naq/item/149-cirrus-logic-cl-gd5464-laguna3d

a lot of technical documentations wow
